sphinxcontrib/drawio.py

```diff
@@ -104,6 +113,14 @@ def render_drawio(self: SphinxTranslator, node: DrawIONode, in_filename: str,
 
     page_index = str(node["config"].get("page-index", 0))
     output_format = node["config"].get("format") or default_output_format
+    if "scale" in node["config"]:
+        scale = str(node["config"].get("scale"))
+    else:
+        scale = str(self.config.drawio_default_scale)
+    if "transparency" in node["config"]:
```

Comment:
Are we not doing the caching as discussed?
There is no reason to do `.get()` if you have checked it is already inside the node.
```py
scale = node["config"].get("scale")
if scale is None:
	scale = self.config.drawio_default_scale
....
"--scale",
str(scale),
```

Ditto for the transparency.